<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetConfig</class>
 <widget class="QWidget" name="SetConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cài đặt thông số</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/newPrefix/pic/logo.ico</normaloff>:/newPrefix/pic/logo.ico</iconset>
  </property>
  <widget class="QPushButton" name="Confirm">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Đóng</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Cổng Arduino</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Chân băng tải</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Cổng cân</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Chân đèn</string>
   </property>
  </widget>
  <widget class="QComboBox" name="scale">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM6</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="conveyor">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="scale_Confirm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
  <widget class="QPushButton" name="arduino_Confirm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conveyor_Confirm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
  <widget class="QPushButton" name="light_Confirm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
  <widget class="QLabel" name="scale_def">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="arduino_def">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="conveyor_def">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="light_def">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="arduino">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM6</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="light">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Chênh lệch cho phép</string>
   </property>
  </widget>
  <widget class="QComboBox" name="range">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="range_Confirm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
